tbx/static_src/sass/components/_team-listing.scss

```diff
@@ -0,0 +1,61 @@
+@use 'config' as *;
+
+.team-listing {
+    display: flex;
+    flex-wrap: wrap;
+    column-gap: $spacer-small;
+    row-gap: $spacer-small-plus;
+
+    @include media-query(large) {
+        max-width: 1270px;
+    }
+
+    &__item {
+        width: 370px;
+
+        @include media-query(medium) {
+            width: calc(33% - #{$spacer-small});
+        }
+
+        @include media-query(large) {
+            width: 230px;
+        }
+    }
+
+    &__image {
+        width: 100%;
+        height: 374px;
+        object-fit: cover;
+        margin-bottom: $spacer-mini;
+
+        @include media-query(medium) {
+            height: auto;
+        }
+
+        @include media-query(large) {
+            height: 230px;
```

Comment:
It only makes a very slight difference - it just ensures it is square rather than slightly off-square if the browser chooses the slightly taller image. However, looking again I think it would be a lot simpler if we just kept the image square at all screen sizes.